Directory.Build.props

```diff
@@ -16,7 +16,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <VersionPrefix>8.1.5</VersionPrefix>
+    <VersionPrefix>8.1.6</VersionPrefix>
```

Comment:
En vez de subir la versión, mejor sube el suffix (`VersionSuffix`) abajo.
Es decir, que quede como `8.1.5-preview-04` 
😊

Reply:
And please remember to update the CHANGELOG.md file as well :)